apps/react/src/challenges/progressbar/ProgressBar.module.css

```diff
@@ -38,6 +38,18 @@
   border: none;
 }
 
+/* Media query for basic mobile view */
+@media screen and (max-width: 480px) {
+  .controls {
```

Comment:
Actually the css in this project is wrotten in a mobile-first manner. Means we use min-width instead of max-width. If you could refactor this media query to use min width then it would be better. Otherwise it is looks alright.

Reply:
i think in either way we need to use max-width property because there is a chance of code redundancy and multiple media queries usage is needed if we need to solve it without using the max-width property. also i need to change the value from 480px to 600px. so , ill do the needful and raise pr again

Reply:
i did the necessary changes and used only min-width can u please go through it